lib/routes/list.js

```diff
@@ -0,0 +1,41 @@
+const { Router } = require("express");
+const router = Router();
+
+const { readLists, readList, addList } = require("../models/list");
+
+router.get("/", async (request, response) => {
+  try {
+    const lists = await readLists();
+    if (!lists) {
+      response.status(404).send(`Lists not found`);
```

Comment:
If you do not return in your if-statement, the following code is executed too.

```suggestion
      return response.status(404).send(`Lists not found`);
```